tests/test_oidc_client.py

```diff
@@ -0,0 +1,22 @@
+from django.test.client import Client
+from django.test.testcases import SimpleTestCase
+
+from mozilla_django_oidc.test import OIDCClient
+
+
+class TestOIDCClient(SimpleTestCase):
+    def test_inherits_from_django_test_client(self):
+        self.assertIsInstance(OIDCClient(), Client)
+
+    def test_oidc_id_token_is_set_to_some_token_by_default(self):
+        self.assertEqual(OIDCClient.oidc_id_token, 'some_oidc_token')
+
+    def test_sets_oidc_id_token_for_session(self):
+        self.assertEqual(OIDCClient().session['oidc_id_token'], 'some_oidc_token')
+
+    def test_sets_specified_id_token_for_session(self):
+        class StubOIDCClient(OIDCClient):
+            oidc_id_token = 'some_other_oidc_token'
```

Comment:
Note that I tested it like this because the session property of the django test client is a little weird to wrap your head around. So instead of instantiating and specifying a new oidc token and refreshing the session, I thought a test like this would be better.